src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
-class Header {
+const f = 'Conduit logo';
+const y = '[Publish comment] button';
 
+class Header {
+  get logo() {
+    return f;
+  };
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the ' + `${f}`;
+  };
 };
 
 class ArticlePage extends PageObject {
-
+  get commentButton() {
+    return y;
+  };
+  clickOnCommentButton() {
+    return 'Click on the ' + y;
+  };
+  assertPageOpened() {
+    return 'The http://test.com is opened';
```

Comment:
I guess we were not supposed to hardcode url

Reply:
agree. ArticlePage should contain assertPageOpened method. The method should return a string 'The ' + url + 'is opened`. You have to use URL property